<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
alternate-background-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 148, 132, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-color: rgb(85, 255, 0);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
background-color: rgb(211, 253, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 255, 255, 0), stop:1 rgba(255, 255, 255, 255));
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="title">
     <string>Providers Manager</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>108</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
     </property>
     <property name="text">
      <string>Last name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>155</y>
       <width>118</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
     </property>
     <property name="text">
      <string>Product type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>202</y>
       <width>63</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
     </property>
     <property name="text">
      <string>Adress</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>249</y>
       <width>164</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
     </property>
     <property name="text">
      <string>Phone number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_last_name_provider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_product_type_provider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adress_provider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>61</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>First name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_first_name_provider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_phone_provider">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_price_provider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_quality_provider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_provider">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>881</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 253, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 255, 255, 0), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_providert">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>580</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>Delete provider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sort_providers">
    <property name="geometry">
     <rect>
      <x>1124</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stat">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>530</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>Statestique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_provider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>Add provider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify_provider">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>Modify provider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_location">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>location of closest provider</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id_modify">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Id </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_provider_5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>590</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Cascadia Code&quot;;
font: 10pt &quot;Cascadia Code&quot;;</string>
    </property>
    <property name="text">
     <string>export pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherche">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>recherche</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="p">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prenom</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>90</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>numero</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>map</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOMNIS_TEMPUS">
    <property name="font">
     <font>
      <family>Cascadia Code</family>
     </font>
    </property>
    <property name="title">
     <string>OMNIS TEMPUS</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menutri">
    <property name="title">
     <string>tri</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuOMNIS_TEMPUS"/>
   <addaction name="menutri"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
